ports/graphics/wayland/dragonfly/patch-src_wayland-server.c

```diff
@@ -0,0 +1,32 @@
+--- src/wayland-server.c.orig	2022-10-31 16:48:01.102621000 +0100
++++ src/wayland-server.c	2022-11-01 14:08:34.644016000 +0100
+@@ -40,7 +40,6 @@
+ #include <assert.h>
+ #include <sys/time.h>
+ #include <fcntl.h>
+-#include <sys/eventfd.h>
+ #include <sys/file.h>
+ #include <sys/stat.h>
+ 
+@@ -1075,10 +1074,6 @@
+ 		return NULL;
+ 	}
+ 
+-	display->terminate_efd = eventfd(0, EFD_CLOEXEC | EFD_NONBLOCK);
```

Comment:
removing that whole block is probably not the best idea ever, but no clue how to replace it
removing it made me remove the err_eventfd too